src/commons/sideContent/__tests__/SideContentCseMachine.tsx

```diff
@@ -26,7 +26,7 @@ test('CSE Machine sets visualization state and renders', async () => {
   expect(screen.queryAllByTestId('sa-cse-machine')).toHaveLength(0);
 
   const context = mockContext();
-  runInContext('const hello="world"; debugger;', context);
+  await runInContext('const hello="world"; debugger;', context);
```

Comment:
This await was remove in #2504 in the file src/commons/sideContent/__tests__/SideContentEnvVisualizer.tsx